### PR TITLE
Fix dmpstore regressed during CodeQL changes

### PR DESCRIPTION
**Pull request by kuqin12**
# Preface

Please ensure you have read the [contribution docs](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md) prior
to submitting the pull request. In particular,
[pull request guidelines](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md#pull-request-best-practices).

## Description

During one of the CodeQL changes, the `dmpstore` command was no longer functional if the input was `dmpstore -all`.

This was caused because the NULL pointer check incorrectly return the function, which is what occurs when using `-all` option. The `ProcessVariables` will properly handle NULL input for `Name`.

For each item, place an "x" in between `[` and `]` if true. Example: `[x]`.
_(you can also check items in the GitHub UI)_

- [x] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [ ] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

This was tested on ARM FVP and QEMU virtual platforms.

## Integration Instructions

N/A

### CODE REVIEW

File: ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c [hunk 1]
```diff
@@ -823,12 +823,6 @@ ShellCommandRunDmpStore (
       // Get the Name of the variable to find
       //
       Name = ShellCommandLineGetRawValue (Package, 1);
-      // MU_CHANGE [BEGIN] - CodeQL change
```
Comment (kuqin12) [inline ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c:826]: This should break the CodeQL, how do we want to handle that?

Comment (os-d) [reply ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c:826]: You can add it to here: https://github.com/microsoft/mu_basecore/blob/release/202208/CodeQlFilters.yml

Comment (makubacki) [reply ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c:826]: Please try to narrow it to the specific rule.

Comment (kuqin12) [reply ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c:826]: Turned out that it was already in the exception list. https://github.com/microsoft/mu_basecore/blob/8b00f3fe8358a019e3325208af5b6f42354eb6ea/CodeQlFilters.yml#LL43C1-L44C1

**Review by makubacki — APPROVED** [review]

**Review by TaylorBeebe — APPROVED** [review]

**Pull request merged**